src/index.ts

```diff
@@ -14,7 +14,11 @@
 
 import TransportStream = require('winston-transport');
 
-import {LOGGING_TRACE_KEY as COMMON_TRACE_KEY, LoggingCommon} from './common';
+import {
+  LOGGING_TRACE_KEY as COMMON_TRACE_KEY,
+  LOGGING_SPAN_KEY as COMMON_SPAN_KEY,
+  LoggingCommon,
```

Comment:
do we also need to introduce **isSampled** field? This way we propagate trace/span only for traces that are sampled.

Reply:
will add shortly